spec/std/http/cookie_spec.cr

```diff
@@ -146,14 +146,34 @@ module HTTP
 
       it "parse max-age as seconds from Time.now" do
         cookie = parse_set_cookie("a=1; max-age=10")
-        delta = cookie.expires.not_nil! - Time.now
-        delta.should be > 9.seconds
-        delta.should be < 11.seconds
+        cookie.max_age.should eq 10.seconds
 
         cookie = parse_set_cookie("a=1; max-age=0")
-        delta = Time.now - cookie.expires.not_nil!
-        delta.should be > 0.seconds
-        delta.should be < 1.seconds
+        cookie.max_age.should eq 0.seconds
+      end
+    end
+
+    describe "expiration_time" do
+      it "by max-age=0" do
+        cookie = parse_set_cookie("bla=1; max-age=0")
+        (cookie.expiration_time.not_nil! <= (Time.now + 1.seconds)).should eq true
+      end
+
+      it "by old date" do
+        cookie = parse_set_cookie("bla=1; expires=Thu, 01 Jan 1970 00:00:00 -0000")
+        cookie.expiration_time.should eq Time.new(1970, 1, 1, 0, 0, 0)
+      end
+
+      it "not expired" do
+        (parse_set_cookie("bla=1; max-age=1").expiration_time.not_nil! > Time.now).should eq true
+      end
+
+      it "not expired" do
+        (parse_set_cookie("bla=1; expires=Thu, 01 Jan 2020 00:00:00 -0000").expiration_time.not_nil! >= Time.new(2020, 1, 1, 0, 0, 0)).should eq true
+      end
+
+      it "not expired" do
```

Comment:
It would be great to add more detail to the spec name, e.g. `without expiration`